routes/weather.ts

```diff
@@ -262,31 +229,26 @@ export const getWateringData = async function( req: express.Request, res: expres
 		wateringData = await weatherProvider.getWateringData( coordinates );
 	}
```

Comment:
I think we should `res.send(err)` and return if `getWateringData` fails as integrity of data is in doubt.

Reply:
We can discuss this in #54